src/main/java/com/parasoft/demoapp/service/GlobalPreferencesService.java

```diff
@@ -264,11 +264,10 @@ private void refreshInventoryDestinations(GlobalPreferencesEntity currentPrefere
             inventoryResponseQueueListener.refreshDestination(currentPreferences.getOrderServiceReplyToQueue());
             inventoryRequestQueueListener.refreshDestination(ActiveMQConfig.getInventoryServiceListenToQueue());
         } else if(MqType.KAFKA == currentPreferences.getMqType()) {
-            String orderServiceRequestTopic = currentPreferences.getOrderServiceRequestTopic();
             MQConfig.currentMQType = MqType.KAFKA;
-            KafkaConfig.setOrderServiceSendToTopic(orderServiceRequestTopic);
-            inventoryRequestTopicListener.refreshDestination(orderServiceRequestTopic);
-            inventoryResponseQueueListener.refreshDestination(currentPreferences.getOrderServiceResponseTopic());
+            KafkaConfig.setOrderServiceSendToTopic(currentPreferences.getOrderServiceRequestTopic());
+            inventoryRequestTopicListener.refreshDestination(currentPreferences.getOrderServiceRequestTopic());
```

Comment:
We need to use `KafkaConfig.DEFAULT_ORDER_SERVICE_REQUEST_TOPIC` due to inventory service always use default destination to listen on.